config/environments/production.rb

```diff
@@ -79,7 +79,7 @@
   # config.logger = ActiveSupport::TaggedLogging.new(Syslog::Logger.new 'app-name')
 
   if ENV["RAILS_LOG_TO_STDOUT"].present?
-    logger           = ActiveSupport::Logger.new(STDOUT)
+    logger           = ActiveSupport::Logger.new($stdout)
```

Comment:
Style/GlobalStdStream: Use `$stdout` instead of `STDOUT`. (https://rubystyle.guide#global-stdout)